public-html/404.php

```diff
@@ -0,0 +1,15 @@
+<?php
+http_response_code(404);
+require $_SERVER['DOCUMENT_ROOT'] . '/../vendor/autoload.php';
+
+use template\Page;
+
+$new = new Page;
+$content = <<<EOT
+<div class="main_content">
```

Comment:
good practice not to use _ in classes

Reply:
Now that is not a change which is feasible now!
because that class is being called in around 14 places
Maybe we will fix em all in a new PR!

Reply:
@AJV009 a simple solution in followup is just add this div inside your page template class since it will be present on all the pages hence you won't need to add it on each page

Reply:
AND yes some new pages and content is also using them!

Reply:
> @AJV009 a simple solution in followup is just add this div inside your page template class since it will be present on all the pages hence you won't need to add it on each page

right but will the template class be flexible anymore?